### PR TITLE
Dev/smart contract

### PR DESCRIPTION
**Pull request by kaikawaliu**

### CODE REVIEW

File: src/libNode/Node.cpp [hunk 1]
```diff
@@ -52,14 +52,13 @@ void addBalanceToGenesisAccount()
 {
     LOG_MARKER();
 
-    const uint256_t bal{100000000000};
+    const uint256_t bal{10000};
```
Comment (KaustubhShamshery) [inline src/libNode/Node.cpp]: Any specific reason to do this ?

Comment (kaikawaliu) [reply src/libNode/Node.cpp]: Will put the number back. Did this change for simplification on local unit test.

**Comment by bb111189** [comment]
@kaikawaliu maybe it will be good to write a brief description of this pr.

**Comment by bb111189** [comment]
Is this PR the same as PR #318 ?

**Comment by Gnnng** [comment]
> Is this PR the same as PR #318 ?

This pr has too much stuff. @kaikawaliu has created a separate one containing the main changes in #318

**Pull request closed without merging**